spec/support/devise.rb

```diff
@@ -4,4 +4,6 @@
 RSpec.configure do |config|
   config.include Devise::Test::ControllerHelpers, type: :controller
   config.include Devise::Test::ControllerHelpers, type: :view
+  config.include Devise::Test::IntegrationHelpers, type: :request
+  config.include Devise::Test::IntegrationHelpers, type: :feature
```

Comment:
Was this was needed for testing the Turbo Frame? I don't think it was needed in other TurboFrame tests?

Reply:
`config.include Devise::Test::IntegrationHelpers, type: :request` was needed because I started to use request specs.

`config.include Devise::Test::IntegrationHelpers, type: :feature` was moved from a different location so that all of the Devise test config was together.